lib/internal/event_target.js

```diff
@@ -219,6 +220,9 @@ class Listener {
 
 function initEventTarget(self) {
   self[kEvents] = new Map();
+  // eslint-disable-next-line no-use-before-define
+  self[kMaxListeners] = NodeEventTarget.defaultMaxListeners;
```

Comment:
This _technically probably works_ since the NodeEventTarget is a subclass but _should_ probably be EventTarget and not NodeEventTarget?

Reply:
Should be, but the `defaultMaxListeners` is a public static that we can't add to `EventTarget`. This works so I suggest we just leave it as is.